proper_paranthetics.py

```diff
@@ -0,0 +1,27 @@
+def fn(s):
+    """
+    shows the state of parenthasies
+    if
+    """
+    s=list(s)
+    s.reverse()
+    n=0
+    while s:
+        t = s.pop()
+        if t == "(": 
+            n += 1
+        elif t == ")": 
+            n -= 1
+
+        if n < 0:
+            return -1
+    if n:
+        return 1
+    else:
+        return 0
+
+def test_fn():
```

Comment:
This needs to be in another file like the other assignments. There are too many cases to test with it just down here at the bottom.